pkg/ip/linuxkit-configure-wireguard-iface.sh

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
+#
+# Copyright (C) 2016-2017 Jason A. Donenfeld <Jason@zx2c4.com>. All Rights Reserved.
+#
+# This file is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2
```

Comment:
We really do not want this to be GPL, we are not planning to ship GPL code in this repo. @zx2c4 can this be relicensed?

Reply:
As this script is a derivative of something from the WireGuard upstream project, it will remain GPLv2. Fortunately, it's easy enough that you could cleanroom re-implement it in a different language if you really want.

But I think for this repo, having a bash script that's GPLv2 shouldn't wind up being a problem for you. It's not as though this is a critical .go file or .c file that's linked into some final executable object. Rather it's just a utility bash script that goes inside images.

Alternatively, you could host it elsewhere -- such as a gist? or something more robust? -- and just wget it in via the package script.